### PR TITLE
Updates language for Resource Packs

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kgarmoe <kgarmoe@chef.io>

### CODE REVIEW

File: docs/shell.md [hunk 1]
```diff
@@ -158,15 +155,15 @@ If you wish to run a single InSpec command and fetch its results, you
...
+$ inspec shell --format json -c 'describe file("/Users/myuser) do it { should exist } end'
+{"version":"0.30.0","profiles":{"":{"supports":[],"controls":{"(generated from in_memory.rb:1 5aab65c33fb1f133d9244017958eef64)":{"title":null,"desc":null,"impact":0.5,"refs":[],"tags":{},"code":"          rule = rule_class.new(id, profile_id, {}) do\n            res = describe(*args, &block)\n          end\n","source_location":{"ref":"/Users/myuser/repo/chef/inspec/lib/inspec/profile_context.rb","line":184},"results":[{"status":"passed","code_desc":"File /Users/myuser should exist","run_time":0.000747,"start_time":"2016-08-16 11:41:40 -0400"}]}},"groups":{"in_memory.rb":{"title":null,"controls":["(generated from in_memory.rb:1 5aab65c33fb1f133d9244017958eef64)"]}},"attributes":[]}},"other_checks":[],"summary":{"duration":0.001078,"example_count":1,"failure_count":0,"skip_count":0}}}
```
Comment (aaronlippold) [inline docs/shell.md]: Would this be more meaningful for the example? Easier to read and also the result from the above example.

```
{
  "version": "1.49.2",
  "controls": [{
    "status": "passed",
    "code_desc": "File /Users/test should exist",
    "run_time": 0.002374,
    "start_time": "2018-01-06 18:32:38 -0500"
  }],
  "other_checks": [],
  "profiles": [{
    "name": "inspec-shell",
    "supports": [],
    "controls": [{
      "title": null,
      "desc": null,
      "impact": 0.5,
      "refs": [],
      "tags": {},
      "code": "",
      "source_location": {
        "ref": "/usr/local/lib/ruby/gems/2.4.0/gems/inspec-1.49.2/lib/inspec/control_eval_context.rb",
        "line": 89
      },
      "id": "(generated from (eval):1 7b6f82c2cc5e4205b3e2c97c8e855f2d)",
      "results": [{
        "status": "passed",
        "code_desc": "File /Users/test should exist",
        "run_time": 0.002374,
        "start_time": "2018-01-06 18:32:38 -0500"
      }]
    }],
    "groups": [{
      "title": null,
      "controls": ["(generated from (eval):1 7b6f82c2cc5e4205b3e2c97c8e855f2d)"],
      "id": "unknown"
    }],
    "attributes": [],
    "sha256": "29c070a90b7e3521babf618215573284a790d92907783d5b2c138f411bfd2e74"
  }],
  "platform": {
    "name": "mac_os_x",
    "release": "17.3.0"
  },
  "statistics": {
    "duration": 0.003171
  }
}
```

File: docs/shell.md [hunk 2]
```diff
@@ -158,15 +155,15 @@ If you wish to run a single InSpec command and fetch its results, you
...
-  ✔  File /Users/ksubramanian should exist
+  ✔  File /Users/myuser should exist
 
 Summary: 1 successful, 0 failures, 0 skipped
 ```
 
 ```bash
-$ inspec shell --format json -c 'describe file("/Users/ksubramanian") do it { should exist } end'
-{"version":"0.30.0","profiles":{"":{"supports":[],"controls":{"(generated from in_memory.rb:1 5aab65c33fb1f133d9244017958eef64)":{"title":null,"desc":null,"impact":0.5,"refs":[],"tags":{},"code":"          rule = rule_class.new(id, profile_id, {}) do\n            res = describe(*args, &block)\n          end\n","source_location":{"ref":"/Users/ksubramanian/repo/chef/inspec/lib/inspec/profile_context.rb","line":184},"results":[{"status":"passed","code_desc":"File /Users/ksubramanian should exist","run_time":0.000747,"start_time":"2016-08-16 11:41:40 -0400"}]}},"groups":{"in_memory.rb":{"title":null,"controls":["(generated from in_memory.rb:1 5aab65c33fb1f133d9244017958eef64)"]}},"attributes":[]}},"other_checks":[],"summary":{"duration":0.001078,"example_count":1,"failure_count":0,"skip_count":0}}}
+$ inspec shell --format json -c 'describe file("/Users/myuser) do it { should exist } end'
```
Comment (aaronlippold) [inline docs/shell.md]: `inspec shell --format json -c 'describe file("/Users/test") do it { should exist } end`

Small typo in the example

File: docs/shell.md (hunk 1 above)
Comment (kagarmoe) [reply docs/shell.md]: I think so! Thanks for all of your thoughtful feedback!

File: docs/shell.md [hunk 3]
```diff
@@ -40,17 +40,14 @@ $ inspec shell -t winrm://UserName:Password@windowsmachine:1234  # Login to wind
...
+## Resource Packs
 
-The InSpec shell may use additional keywords provided in resource packs.
-A resource pack is a profile that defines new language terms that can
-be used in InSpec. For example, the profile in `examples/profile` in
-the InSpec git repo defines a `gordon_config` resource. To use these
-resources with the InSpec shell, you will need to download and specify
-them as a dependency.
+Use resource packs to share custom resources with other InSpec users.
+A resource pack is an inspec profile that contains only custom resources and no other controls or tests.
```
Comment (jerryaldrichiii) [inline docs/shell.md]: Should this be `InSpec profile`?

File: docs/shell.md [hunk 4]
```diff
@@ -40,17 +40,14 @@ $ inspec shell -t winrm://UserName:Password@windowsmachine:1234  # Login to wind
...
-be used in InSpec. For example, the profile in `examples/profile` in
-the InSpec git repo defines a `gordon_config` resource. To use these
-resources with the InSpec shell, you will need to download and specify
-them as a dependency.
+Use resource packs to share custom resources with other InSpec users.
+A resource pack is an inspec profile that contains only custom resources and no other controls or tests. 
 
-To use the `gordon_config` resource that is provided in the `examples/profile`
-in the InSpec repo you can run the following:
+For example, the profile in [`examples/profile`](https://github.com/chef/inspec/tree/master/examples/profile)in the InSpec git repo defines a [`gordon_config` resource](https://github.com/chef/inspec/blob/master/examples/profile/controls/gordon.rb). To use these resources within the InSpec shell, you will need to download and specify them as a dependency.
```
Comment (jerryaldrichiii) [inline docs/shell.md]: Do we have this documented anywhere?

> you will need to download and specify them as a dependency.

File: docs/shell.md [hunk 5]
```diff
@@ -40,17 +40,14 @@ $ inspec shell -t winrm://UserName:Password@windowsmachine:1234  # Login to wind
...
+
+Once you have local access to the profile, you can use the `gordon_config` custom resource provided in the `examples/profile` Github repo in your local environment :
```
Comment (jerryaldrichiii) [inline docs/shell.md]: Could we change this to `GitHub`?

File: docs/shell.md [hunk 6]
```diff
@@ -158,15 +155,61 @@ If you wish to run a single InSpec command and fetch its results, you
 may use the `-c` flag. This is similar to using `bash -c`.
 
 ```bash
-$ inspec shell -c 'describe file("/Users/ksubramanian") do it { should exist } end'}
+$ inspec shell -c 'describe file("/Users/myuser") do it { should exist } end'}
```
Comment (jerryaldrichiii) [inline docs/shell.md]: Need to remote the ending `}`.

**Review by jerryaldrichiii — DISMISSED** [review]
Great work! Only a few suggestions/questions.

File: docs/shell.md (hunk 4 above)
Comment (kagarmoe) [reply docs/shell.md]: In the profile & profile inheritance section

File: docs/shell.md (hunk 5 above)
Comment (kagarmoe) [reply docs/shell.md]: done

**Review by jerryaldrichiii — APPROVED** [review]

**Review by jquick — APPROVED** [review]
Thanks @kagarmoe !

**Pull request merged**